vectordb_bench/backend/clients/pgvector/cli.py

```diff
@@ -61,11 +70,45 @@ class PgVectorTypedDict(CommonTypedDict):
         Optional[str],
         click.option(
             "--quantization-type",
-            type=click.Choice(["none", "halfvec"]),
+            type=click.Choice(["none", "bit", "halfvec"]),
             help="quantization type for vectors",
             required=False,
         ),
     ]
+    reranking: Annotated[
+        Optional[bool],
+        click.option(
+            "--reranking/--skip-reranking",
+            type=bool,
+            help="Enable reranking for HNSW search for binary quantization",
+            default=False,
+        ),
+    ]
+    reranking_metric: Annotated[
+        Optional[str],
+        click.option(
+            "--reranking-metric",
+            type=click.Choice(
+                [metric.value for metric in MetricType if metric.value not in ["HAMMING", "JACCARD"]]
+            ),
+            help="Distance metric for reranking",
+            default="COSINE",
```

Comment:
@Sheharyar570 this shouldn't be COSINE..

Reply:
reranking_metric that's used to calculate the distance between actual vectors in the reranking query. We can't use bit distance operator i.e. hamming, jaccard etc for this operation. 

Hamming or jaccard selected based on the dataset information, if dataset metric is not defined and bit quantization is selected it will self.metric will return hamming_bit_ops or <~>

Reply:
@Sheharyar570 oh okay go it. However, in that case do we need to limit the distance to anything other than Hamming and Jaccard? I mean it can also be Jaccard or Hamming? 
Talking about `[metric.value for metric in MetricType if metric.value not in ["HAMMING", "JACCARD"]]` 4 lines above.